RELEASE.md

```diff
@@ -33,7 +33,7 @@ For the actively supported versions as of December 2023, the following release c
 | Python version | Currently supported | Scheduled/expected end of active support |
 | -------------- | ------------------- | ---------------------------------------- |
 | Python 3.8     | &#10060;            | 10-January-2024 (dropped)                |
-| Python 3.9     | &#9989;             | July 2024 (expected)                     |
+| Python 3.9     | &#10060;            | 10-July-2024 (dropped)                    |
```

Comment:
we can maybe change this as scheduled
```suggestion
| Python 3.9     | &#10060;            | 10-July-2024 (scheduled)                  |
```
and then do a follow-up PR that changes it back to `dropped`.

Not really necessary but may be useful to our users

Reply:
Then we'll have to merge it now ;)